frontend/pages/play/index.vue

```diff
@@ -25,14 +28,15 @@ function navigateToTeamname() {
       />
       <UIcon
         name="ic:round-check"
-        class="absolute right-14 top-1/2 -translate-y-1/2 transform p-12 text-sc-green"
-        v-if="roomCode"
+        class="absolute right-14 top-1/2 -translate-y-1/2 transform p-12 text-sc-green transition-opacity"
+        :class="{ 'opacity-0': !teamName }"
```

Comment:
ich denke mal, das war nicht mit prüfen auf teamName gedacht :)
```suggestion
        :class="{ 'opacity-0': !roomCodeIsValid }"
```